src/daemons/rhsmcertd.c

```diff
@@ -86,33 +85,24 @@ static LOG_LEVEL log_level = DEFAULT_LOG_LEVEL;
 static gboolean show_debug = FALSE;
 static gboolean run_now = FALSE;
 static gint arg_cert_interval_minutes = -1;
-static gint arg_heal_interval_minutes = -1;
 static gint arg_reg_interval_minutes = -1;
 static gboolean arg_no_splay = FALSE;
 static gboolean arg_auto_registration = FALSE;
 static int fd_lock = -1;
 
 struct CertCheckData {
     int interval_seconds;
-    bool heal;
     char *next_update_file;
 };
 
 static GOptionEntry entries[] = {
     /* marked deprecated as of 02-19-2013, needs to be removed...? */
-    {"cert-interval", 0, 0, G_OPTION_ARG_INT, &arg_heal_interval_minutes,
+    {"cert-interval", 0, 0, G_OPTION_ARG_INT, &arg_cert_interval_minutes,
      N_("deprecated, see --cert-check-interval"),
      "MINUTES"},
```

Comment:
`--cert-interval` is an alias for `--heal-interval`, so simply drop it; yes, the help text here is wrongly pointing to `--cert-check-interval`, even if the actual variable set is the one for `--heal-interval`

Reply:
Yeah, If anything has been deprecated since 2013, then we can drop it now.

Reply:
Addressed by commit: [e4de3f4](https://github.com/candlepin/subscription-manager/pull/3455/commits/e4de3f402334d657c10ac83d2bafceeb5874d1fa)